src/tests/ftest/slurm_setup.py

```diff
@@ -184,11 +183,17 @@ def start_munge(args):
     if execute_cluster_cmds(args.control, ["; ".join(cmd_list)]) > 0:
         return 1
     # remove any existing key from other nodes
-    cmd_list = ["{} rm -f /etc/munge/munge.key".format(sudo),
-                "scp -p {}:/etc/munge/munge.key /etc/munge/munge.key".format(
-                    args.control)]
+    cmd_list = ["{} rm -f /etc/munge/munge.key".format(sudo)]
     if execute_cluster_cmds(nodes, ["; ".join(cmd_list)]) > 0:
         return 1
+
+    # copy munge.key to all hosts
+    command = "{0} --copy {1} --dest {1}".format(get_clush_command(nodes), "/etc/munge/munge.key")
+    try:
+        run_command(command)
+    except DaosTestError:
+        return 1
```

Comment:
Do we have enough logging to figure out the problem if this fails?

Reply:
Yes, we call avocado.utils.process.run with verbose=True